packages/logger/src/createLogChannel.ts

```diff
@@ -5,50 +5,20 @@ export const createLogChannel: LogChannelFactory = (
   channel: LogChannelName
 ) => ({
   debug: (message, ...context) =>
-    createLoggable<LogSeverity.DEBUG>(
-      LogSeverity.DEBUG,
-      channel,
-      message,
-      context
-    ),
+    createLoggable(LogSeverity.DEBUG, channel, message, context),
```

Comment:
`createLoggable` has a generic parameter that can be inferred by Typescript by the first parameter.
Check it out:
<img width="517" alt="Screen Shot 2020-12-15 at 1 05 54 PM" src="https://user-images.githubusercontent.com/11247544/102254187-541c5b00-3ed6-11eb-9590-ac24c36ed465.png">